### PR TITLE
Update openapi-aot.md

### LINKED ISSUE #34340: Update OpenApi supports trimming and Native AOT
### Description

[Enter feedback here]

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-9.0?view=aspnetcore-9.0&branch=pr-en-us-34320#openapi

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/release-notes/aspnetcore-9.0.md

### Document ID

4e75ad25-2c3f-b28e-6a91-ac79a9c683b6

### Article author

@Rick-Anderson

[Related Issues](https://github.com/dotnet/AspNetCore.Docs/issues?q=is%3Aissue+is%3Aopen+4e75ad25-2c3f-b28e-6a91-ac79a9c683b6)

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #34340

### CODE REVIEW

File: aspnetcore/release-notes/aspnetcore-9/includes/openapi-aot.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
 ### Microsoft.AspNetCore.OpenApi supports trimming and Native AOT
 
-The new built-in OpenAPI support in ASP.NET Core now also supports trimming and Native AOT.
-
-### Get started
+The new built-in OpenAPI in ASP.NET Core supports trimming and Native AOT.
```
Comment (Rick-Anderson) [inline aspnetcore/release-notes/aspnetcore-9/includes/openapi-aot.md:3]:
```suggestion
The new built-in OpenAPI in ASP.NET Core supports trimming and Native AOT. The following steps create and publish and OpenAPI app with trimming and Native AOT:
```

File: aspnetcore/release-notes/aspnetcore-9/includes/openapi-aot.md [hunk 2]
```diff
@@ -1,8 +1,6 @@
 ### Microsoft.AspNetCore.OpenApi supports trimming and Native AOT
 
-The new built-in OpenAPI support in ASP.NET Core now also supports trimming and Native AOT.
-
-### Get started
+The new built-in OpenAPI in ASP.NET Core supports trimming and Native AOT. The following steps create and publish and OpenAPI app with trimming and Native AOT:
```
Comment (Rick-Anderson) [inline aspnetcore/release-notes/aspnetcore-9/includes/openapi-aot.md:3]:
```suggestion
The new built-in OpenAPI in ASP.NET Core supports trimming and Native AOT. The following steps create and publish an OpenAPI app with trimming and Native AOT:
```

File: aspnetcore/release-notes/aspnetcore-9/includes/openapi-aot.md [hunk 3]
```diff
@@ -1,8 +1,6 @@
 ### Microsoft.AspNetCore.OpenApi supports trimming and Native AOT
 
-The new built-in OpenAPI support in ASP.NET Core now also supports trimming and Native AOT.
-
-### Get started
+The new built-in OpenAPI in ASP.NET Core supports trimming and Native AOT. The following steps create and publish an OpenAPI app with trimming and Native AOT:
```
Comment (Rick-Anderson) [inline aspnetcore/release-notes/aspnetcore-9/includes/openapi-aot.md:3]:
```suggestion
Trimming and Native AOT are now available for OpenAPI in ASP.NET Core. The following steps create and publish an OpenAPI app with trimming and Native AOT:
```

File: aspnetcore/release-notes/aspnetcore-9/includes/openapi-aot.md [hunk 4]
```diff
@@ -1,8 +1,6 @@
 ### Microsoft.AspNetCore.OpenApi supports trimming and Native AOT
 
-The new built-in OpenAPI support in ASP.NET Core now also supports trimming and Native AOT.
-
-### Get started
+Trimming and Native AOT are now available for OpenAPI in ASP.NET Core. The following steps create and publish an OpenAPI app with trimming and Native AOT:
```
Comment (tdykstra) [inline aspnetcore/release-notes/aspnetcore-9/includes/openapi-aot.md:3]:
```suggestion
OpenAPI in ASP.NET Core supports trimming and native AOT. The following steps create and publish an OpenAPI app with trimming and Native AOT:
```

**Review by tdykstra — APPROVED** [review]
Looks good, one suggestion.

**Pull request merged**